projects/plugins/boost/app/assets/src/js/utils/pricing.ts

```diff
@@ -0,0 +1,4 @@
+import { derived } from 'svelte/store';
+import config from '../stores/config';
+
+export const pricing = derived( config, $config => $config.pricing );
```

Comment:
I've removed the util completely in https://github.com/Automattic/jetpack/pull/29250/commits/04849cfb6cbec484db32e9b6a8ee4090b2b10fc0.

Reply:
Cool beans. This is completely optional, but did you know in Svelte you can create the same effect as defining your derived store in your `<script>` block by using a `$` block like so:

```
$: pricing = $config.pricing;
```

Svelte is smart enough to automatically imply an invisible `let pricing;` above that. Every time `$config` changes, `pricing` will be updated too. :)

Or, alternately:
```
$: currencyObjectAfter = getCurrencyObject(
  $config.pricing.yearly.priceAfter,
  $config.pricing.yearly.currencyCode
);
```

This will re-run the `$` block every time `$config` is changed :)

All of this is just suggested extra learning. I think this is acceptable as-is.

Reply:
Thank you for the suggestions!

I took the advice and made the improvements in fcb058f1096cd630f158b72528d3ba21e3eb30bf.